src/-make-function.js

```diff
@@ -71,6 +80,7 @@ var makeFunction = {
 	// `construct` is called when the `new` operator is used.
 	// It needs to return an observable instance.
 	construct: function(target, argumentsList, newTarget) {
+		//debugger;
```

Comment:
can you remove this?

Reply:
Yeah, I have other bugs, will remove this in those.